packages/app/src/subpages/notebookSelector/components/Notebook.tsx

```
@@ -0,0 +1,51 @@
+  onClick: () => void;
+}) {
+  return (
+    <button
+      className="border border-border group hover:bg-background-2 min-h-16 rounded-lg overflow-hidden flex duration-200 cursor-pointer w-full"
+      onClick={onClick}
+    >
+      <div className="w-1 bg-border duration-200"></div>
+      <div className="py-2.5">
+        <div className="flex items-center">
+          {nb.icon ? (
+            <img src={nb.icon} className="ml-3 mr-2 h-6 rounded-sm" />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)